pmm-app/src/shared/components/helpers/api.ts

```diff
@@ -42,21 +38,21 @@ export class ApiRequest {
       });
   }
 
-  async delete<T>(path: string): Promise<void | T> {
+  async delete<T>(path: string): Promise<T> {
     return this.axiosInstance
       .delete<T>(path)
       .then((response): T => response.data)
-      .catch((): void => {
-        // Notify.error(e.message);
+      .catch((e) => {
+        throw e;
       });
   }
 
-  async patch<T, B>(path: string, body: B): Promise<void | T> {
+  async patch<T, B>(path: string, body: B): Promise<T> {
     return this.axiosInstance
       .patch<T>(path, body)
       .then((response): T => response.data)
-      .catch((): void => {
-        // Notify.error(e.message);
+      .catch((e) => {
+        throw e;
```

Comment:
Omitting these two lines of code leads to the same result as leaving them here, so the question is: what additional value does it bring?

Reply:
Right! Fixed by show the error notification there to make it consistent with the other functions. Ofc deleting the catch was possible too but solved this way to still get notifications, thanks!!